examples/kubernetes/1.13/cilium.yaml

```diff
@@ -292,8 +292,15 @@ spec:
       hostNetwork: true
       hostPID: false
       initContainers:
-      - command:
-        - /init-container.sh
+      - command: ['sh', '-c',
+            'if [ "${CLEAN_CILIUM_STATE}" = "true" ]; then
```

Comment:
The environment variable `CLEAN_CILIUM_STATE` and `CLEAN_CILIUM_BPF_STATE` can be assigned to the `--force` and `--bpf-state-only` with viper. so we can call `cilium cleanup` instead of detecting if the environment variable is set here.

Reply:
Oh cool, I'll look into this!